core/controllers/base_test.py

```diff
@@ -100,7 +100,7 @@ def test_that_no_get_results_in_500_error(self):
             # Some of these will 404 or 302. This is expected.
             response = self.testapp.get(url, expect_errors=True)
             if response.status_int not in [200, 302, 400, 401, 404]:
-                print url
+                print url # disable-bad-pattern-check
```

Comment:
Actually I think we can remove this and include it in the optional msg argument of the self.assertIn() call below.